index.html

```diff
@@ -0,0 +1,208 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navmenu">
+      <h3>My Logo</h3>
+    </nav>
+    <img src="image/Union.png" alt="" />
+
+  </header>
+  <section class="backgroundimg">
+    <div class="menuSection">
+      <h1 class="introduction">I’m Mariane Glad to see you!</h1>
+      <p class="Whoiam">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+    </div>
+    <div class="menuSection">
+      <a href="#" target="_blank">LET'S CONNECT</a>
+      <ul class="socialMedia">
+        <li>
+          <img src="image/Twitter.png" alt="" />
+
+        </li>
+        <li>
+          <img src="image/Linkedin.png" alt="" />
+
+        </li>
+        <li>
+          <img src="image/Meduim.png" alt="" />
+
+        </li>
+        <li>
+          <img src="image/Github.png" alt="" />
+
+        </li>
+        <li>
+          <img src="image/angellist.png" alt="" />
+
+        </li>
+      </ul>
+    </div>
+  </section>
+  <div id="cards">
+
+    <div id="flex">
+      <div class="card">
+        <img class="img" src="image/card-1.svg" alt="">
+        <div class="content">
+          <div class="topic">Tonic</div>
+          <div class="sub-topic">
+            <span>CANOPY</span>
+            <span class="dot"></span>
+
+            <span>Back End Dev</span>
+            <span class="dot"></span>
+
+            <span>2015</span>
+          </div>
+          <div class="description">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div class="button-links">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="project">See Project</div>
+        </div>
+      </div>
+
+      <div class="card">
+        <img class="img" src="image/card-2.svg" alt="">
+        <div class="content">
+          <div class="topic">Multi-Post <br /> Stories</div>
+          <div class="sub-topic">
+            <span>CANOPY</span>
+            <span class="dot"></span>
+
+            <span>Back End Dev</span>
+            <span class="dot"></span>
+
+            <span>2015</span>
+          </div>
+          <div class="description">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div class="button-links">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="project">See Project</div>
```

Comment:
- [ ] Kindly ensure you use a `<button>` tag or an anchor tag `<a>` to wrap the "See Project" text because it leads to another page or popup to display the project and because of that, it should be treated as a link or a button 😎